addons/mass_mailing_sms/views/mailing_mailing_views.xml

```diff
@@ -60,113 +58,258 @@
+                            <span name="next_departure_text" attrs="{'invisible': [('mailing_type', '!=', 'sms')]}">This SMS marketing is scheduled for </span>
+                            <field name="next_departure" class="oe_inline"/>.
+                        </strong>
+                    </div>
+                </div>
 
-            <xpath expr="//field[@name='contact_list_ids']" position="attributes">
-                <attribute name="context">
-                    {'mailing_sms' : context.get('mailing_sms')}
-                </attribute>
-            </xpath>
-            <!-- Option page tweaks -->
-            <xpath expr="//field[@name='email_from']" position="attributes">
-                <attribute name="attrs">{
-                    'invisible': [('mailing_type', '!=', 'mail')],
-                    'readonly': [('state', 'in', ('sending', 'done'))]}
-                </attribute>
-            </xpath>
-            <xpath expr="//label[@for='reply_to']" position="attributes">
-                <attribute name="attrs">{'invisible': [('mailing_type', '!=', 'mail')]}</attribute>
-            </xpath>
-            <xpath expr="//div[@name='reply_to_details']" position="attributes">
-                <attribute name="attrs">{'invisible': [('mailing_type', '!=', 'mail')]}</attribute>
-            </xpath>
-            <xpath expr="//label[@for='attachment_ids']" position="attributes">
-                <attribute name="attrs">{'invisible': [('mailing_type', '!=', 'mail')]}</attribute>
-            </xpath>
-            <xpath expr="//div[@name='attachment_ids_details']" position="attributes">
-                <attribute name="attrs">{'invisible': [('mailing_type', '!=', 'mail')]}</attribute>
-            </xpath>
-            <xpath expr="//field[@name='mail_server_id']" position="attributes">
-                <attribute name="attrs">{'invisible': ['|', ('mailing_type', '!=', 'mail'),
-                    ('mail_server_available', '=', False)], 'readonly': [('state', 'in', ('sending', 'done'))]}</attribute>
-            </xpath>
-            <xpath expr="//field[@name='unique_ab_testing']" position="attributes">
-                <attribute name="attrs">{'invisible': [('mailing_type', '!=', 'mail')]}</attribute>
-            </xpath>
-            <xpath expr="//div[@name='contact_ab_pc']" position="attributes">
-                <attribute name="attrs">{'invisible': [('mailing_type', '!=', 'mail')]}</attribute>
-            </xpath>
-            <xpath expr="//label[@for='contact_ab_pc']" position="attributes">
-                <attribute name="attrs">{'invisible': [('mailing_type', '!=', 'mail')]}</attribute>
-            </xpath>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button name="action_view_delivered"
+                            id="button_view_delivered"
```

Comment:
Same here, some attributes could be grouped on a line to lessen diff and file size (e.g. generally, name + id and type + class)